source/dlangbot/cron.d

```diff
@@ -138,10 +138,11 @@ mergable:
 auto detectPRWithPersistentCIFailures(PRTuple t)
 {
     // label PR with persistent CI failures
-    auto status = t.pr.status;
-    auto failCount = status.filter!((e){
-        if (e.state == GHCiStatus.State.failure ||
-            e.state == GHCiStatus.State.error)
+    // TODO: unclear whether we want all statuses for PR commit, or only the latest
+    auto status = t.pr.combinedStatus;
```

Comment:
This will be yet another request for the cron job. Each request multiplies with ~400 here.

Reply:
Only open auto-merge PRs, not so many.